docs/troubleshooting.md

```diff
@@ -0,0 +1,30 @@
+# Development troubleshooting
+
+## Commit error: Husky not found
+
+You might need to add these lines to the start of the untracked `/.husky/_/husky.sh` file:
+
+```bash
+export NVM_DIR="$HOME/.nvm"
+[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"  # This loads nvm
+[ -s "$NVM_DIR/bash_completion" ] && \. "$NVM_DIR/bash_completion"  # This loads nvm bash_completion
+```
+
+## Docker error: No space left on device
```

Comment:
It may also be worth mentioning that we have two commands to stop the containers.
1. `npm run dev:down`: This command will remove the unused volumes
2. `npm run dev:stop`: This command stop the containers but keeps the volumes 

Because we are using the command `docker-compose up` when we run the command `npm rund dev` in the terminal. I suggest using the command `npm run dev:down` to stop the container.  I think we may need to add some logic to decide whether we run the `docker-compose up` or the `docker-compose start` command every time we want to run the app so that, we do not keep unnecessary volumes in our computers. But for now, running the `npm run dev:down` will keep our computers a bit cleaner.

Reply:
Do you have a suggestion on how to formulate this here? At least on OSX, running `yarn dev:down` doesn't solve the space issue (or is it maybe another `npm/yarn` issue…).

Reply:
Yes, if the volumes are already created, you need to prune. My comment is more about adding some mentions to the two commands to stop the volumes:

1. `npm run dev:down`: This command will remove the unused volumes
2. `npm run dev:stop`: This command stop the containers but keeps the volumes